delivery_package_number/models/stock_picking.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    number_of_packages = fields.Integer(
+        string="Number of Packages",
+        default=1,
+        copy=False,
+    )
+
+
+class StockMoveLine(models.Model):
+    _inherit = "stock.move.line"
+
+    @api.onchange("result_package_id")
+    def onchange_package_ids(self):
+        """TODO: in v13 Change field to compute readonly=False"""
+        picking = self.picking_id
+        picking.number_of_packages = len(picking.package_ids) or 1
+
+    def write(self, vals):
+        """TODO: in v13 Change field to compute readonly=False"""
```

Comment:
can we take the opportunity of this migration to handle this TODO?

Reply:
I assume it means turning the field into a stored compute with readonly=False so its editable?

Reply:
Yes, that's it

Reply:
Do I need to preserve this funcionality? `Now we put in pack some quantities and the number of packages is set as the number of packs is added to the number already set`
It's a bit convoluted to do in a compute

Reply:
OK, we can keep the TODO for me.